src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,21 @@
-export const TodoInfo = () => {};
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+interface Props {
+  todo: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const { completed, title, user } = todo;
+
+  return (
+    <article
+      data-id={todo.id}
+      className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}
```

Comment:
mb it would be better to use the ClassName library here? Or you can use operator '&&'

Reply:
Good point, use `classnames`